modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/test/java/com/liferay/portal/workflow/kaleo/runtime/internal/assignment/AggregateKaleoTaskAssignmentSelectorImplTest.java

```diff
@@ -144,30 +162,35 @@ private KaleoTaskAssignment _createKaleoTaskAssignment(
 	private void _setUpAggregateKaleoTaskAssignmentSelectorImpl()
 		throws Exception {
 
-		KaleoTaskAssignmentSelectorRegistryImpl
-			kaleoTaskAssignmentSelectorRegistryImpl =
-				new KaleoTaskAssignmentSelectorRegistryImpl();
+		_kaleoTaskAssignmentSelectorRegistryImpl =
+			new KaleoTaskAssignmentSelectorRegistryImpl();
 
 		for (Map.Entry<String, List<KaleoTaskAssignment>> entry :
```

Comment:
@W2jx, please move new KaleoTaskAssignmentSelectorRegistryImpl() to field initialized. It will keep consistent with _aggregateKaleoTaskAssignmentSelectorImpl field.